modules/openapi-generator/src/test/resources/3_0/helidon/petstore-for-testing.yaml

```diff
@@ -72,7 +72,10 @@ paths:
             - 'write:pets'
             - 'read:pets'
       requestBody:
-        $ref: '#/components/schemas/Pet'
+        content:
```

Comment:
FYI. @tjquinno  there are changes to helidon test spec and files

Reply:
Yes because the spec was no valid. I've fixed another one in samples.